src/webapp-lib/doc/features.pug

```diff
@@ -60,10 +60,10 @@ block content
             Read more about it in the #[a(href="https://doc.cocalc.com/x11.html") X11 documentation].
 
         li
-          a(href="linux-terminal.html") Linux Terminal:
+          a(href="linux.html") Online Linux Environment:
```

Comment:
Why make this change?  I know "Linux Environment" is technically more general.  However, it gets an **order of magnitude** less searches on Google than "Linux Terminal".    People are 10x more likely to search for "linux terminal" than "linux environment".    What's the logic here?